app/javascript/gobierto_data/webapp/components/Dataset.vue

```diff
@@ -451,7 +449,7 @@ export default {
       this.isPublicVizLoading = false
     },
     async getPrivateVisualizations() {
-      const userId = getUserId()
+      const userId = Number(getUserId());
```

Comment:
Keep in mind if `userId` was `0` the conditional clause would be falsy, so you wouldn't call to getVisualizations.
By the way, another "cool" way to cast an integer is: `      const userId = +getUserId();`

Reply:
Yes, if the `user` was `0` = user is not logged, so he can't see privateVisualizations.

Thanks for the tip!